lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -367,7 +367,7 @@ module.exports = class MethodMappings {
         },
 
         async locateMultipleElements(element) {
-          const locator = Locator.create(element);
+          const locator = Locator.create(element, this.transport.api.isAppiumClient());
```

Comment:
```suggestion
	const {isAppiumClient} = this.transport.api;
          const locator = Locator.create(element, isAppiumClient());
```

Reply:
Cannot do this, because if we destructure the definition of `isAppiumClient` then `this` used inside `isAppiumClient` method will become `undefined` unless we bind the method again with `this.transport.api`.